### PR TITLE
Add Multi-Region Storefront Article to BC4WP

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-978](https://jira.bigcommerce.com/browse/DEVDOCS-978)

## What changed?

Add Multi-Region Storefront Article to BC4WP

### CODE REVIEW

**Comment by aglensmith** [comment]
Gotta fix title and otp, one sec.

**Comment by tatiana-perry** [comment]
@karen-white 🙇‍♀

File: markdown/bigcommerce-for-wordpress/extending-the-plugin/multi-region-storefront.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+### On This Page
+
+* [The BigCommerce Approach to Serving Multiple Regions](#the-bigCommerce-approach-to-serving-multiple-regions)
+* [Primary Use Cases](#primary-use-cases)
+* [Multi-Region Implementation Examples](#multi-region-implementation-examples)
+
+</div>
+
+This article demonstrates some BigCommerce for WordPress plugin Multi-Channel capabilities and provides an example to illustrate how a multi-region storefront can be setup with minimal code.
```
Comment (karen-white) [inline markdown/bigcommerce-for-wordpress/extending-the-plugin/multi-region-storefront.md]: setup should be set up

Comment (aglensmith) [reply markdown/bigcommerce-for-wordpress/extending-the-plugin/multi-region-storefront.md]: replaced

**Comment by karen-white** [comment]
@amckemie @becomevocal It would be good to get your thoughts on this too. Is there anything that we want to cover in the documentation for BC4WP channel capabilities that weren't in the blog post this was derived from?

**Comment by karen-white** [comment]
I'm not sure I love pulling the documentation word for word from the blog post, although I do think it makes sense to reference it as an additional resource. Can we discuss reworking the approach?

**Comment by becomevocal** [comment]
@karen-white makes sense to be if we highlight only the aspects of how you trigger the channel in the code and the sample plugins. The blog can be linked out to for more context on the approach.

**Pull request closed without merging**